clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/DetachInstancesAtomicOperation.groovy

```diff
@@ -78,9 +78,9 @@ class DetachInstancesAtomicOperation implements AtomicOperation<Void> {
         return false
       }
 
-      if ((autoScalingGroup.desiredCapacity - validInstanceIds.size()) < autoScalingGroup.minSize) {
+      int newMin = autoScalingGroup.desiredCapacity - validInstanceIds.size()
+      if (description.decrementDesiredCapacity && newMin < autoScalingGroup.minSize) {
```

Comment:
Might need some default valuing for `decrementDesiredCapacity`?

Default to `true` if `null`?

Reply:
doesn't it default to `false` by virtue of being a `boolean`?

Reply:
Totally correct, I glossed over that.

👍